### ISSUE TITLE
Order of warping image and reference image

### ISSUE THREAD

**Issue by fcbfcb1998** [liudakai2/UnsupDIS-pytorch#36]
Did you change the order of training and testing for UDIS-D(order of target image and warping image)? I found that the order of both coarse stitching and overlapping PSNR and SSIM are differnet from original implementation.

**Comment by LiYQi** [comment]
I have the same problem.

**Comment by liudakai2** [comment]
Yes. In practical scenarios (e.g., two images captured by two cameras simultaneously), the warping order does not matter. We are merely used to warping left-image to right-image, not the other way around (in the original implementation).

If you want to follow the original setting, just switch the data (image and mask) order manually in the code.
